### PR TITLE
src: lib: Add support to fn_c and fn_py

### LINKED ISSUE #17: Cannot use slice as input
```
error[E0277]: the trait bound `&[u8; 3]: PyClass` is not satisfied
   --> src/lib.rs:94:1
    |
94  | / export_cpy!(
95  | |     mod navigator {
96  | |         enum AdcChannel {
97  | |             Ch0,
...   |
251 | |     }
252 | | );
    | |_^ the trait `PyClass` is not implemented for `&[u8; 3]`
```

### PR DESCRIPTION
**Pull request by patrickelectric**
Signed-off-by: Patrick José Pereira <patrickelectric@gmail.com>

Fix #17

### CODE REVIEW

**Comment by patrickelectric** [comment]
ping @joaoantoniocardoso @RaulTrombin

File: src/lib.rs [hunk 1]
```diff
@@ -33,6 +33,14 @@ macro_rules! export_cpy {
         export_cpy!(@generate_function $name($($param : $ptype),*) $(-> $ret)? $body);
         export_cpy!(@process_item $($rest)*);
     };
+    (@process_item fn_c $name:ident($($param:ident : $ptype:ty),*) $(-> $ret:ty)? { $($body:tt)* } $($rest:tt)*) => {
+        export_cpy!(@generate_c_function $name($($param : $ptype),*) $(-> $ret)? { $($body)* });
```
Comment (RaulTrombin) [inline src/lib.rs]:
```suggestion
        export_cpy!(@generate_c_function $name($($param : $ptype),*) $(-> $ret)? $($body)*);
```

File: src/lib.rs [hunk 2]
```diff
@@ -33,6 +33,14 @@ macro_rules! export_cpy {
...
+        export_cpy!(@process_item $($rest)*);
+    };
+    (@process_item fn_py $name:ident($($param:ident : $ptype:ty),*) $(-> $ret:ty)? { $($body:tt)* } $($rest:tt)*) => {
+        export_cpy!(@generate_py_function $name($($param : $ptype),*) $(-> $ret)? { $($body)* });
```
Comment (RaulTrombin) [inline src/lib.rs]:
```suggestion
        export_cpy!(@generate_py_function $name($($param : $ptype),*) $(-> $ret)? $($body)*);
```

**Review by RaulTrombin — CHANGES_REQUESTED** [review]

**Comment by RaulTrombin** [comment]
seems ok!
image

**Review by RaulTrombin — APPROVED** [review]

**Pull request merged**